src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageDetailsTabViewModel.cs

```diff
@@ -38,8 +39,9 @@ public async Task InitializeAsync(DetailControlModel detailControlModel, INuGetP
         {
             var nuGetFeatureFlagService = await ServiceLocator.GetComponentModelServiceAsync<INuGetFeatureFlagService>();
             _readmeTabEnabled = await nuGetFeatureFlagService.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.RenderReadmeInPMUI);
-
-            ReadmePreviewViewModel = new ReadmePreviewViewModel(nugetPackageFileService, currentFilter, _readmeTabEnabled);
+#pragma warning disable CS0618 // Type or member is obsolete
+            ReadmePreviewViewModel = new ReadmePreviewViewModel(nugetPackageFileService, currentFilter, new PreviewBuilder().Build(), _readmeTabEnabled);
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
which API is obsolete, and why are we using it if it's marked as obsolete?

I accept suppressing these obsolete messages when we mark something in our own code as `[Obsolete]`, and need to maintain backwards compatibility until we delete the obsolete API.  But using a known obsolete API from a dependency seems risky.  If they delete or break the API, we don't have tests running in the VS repo to inform the owner of the API that they broke NuGet's functionality.

Reply:
PreviewBuilder is marked as obsolete because it hasn't been finalized. It's owned by the editor team and they're aware we're using this package and said they would let us know if the API changes.